For Xgo Run

joeykchen opened the pull request.

JiepengTan commented:
@joeykchen   Better to integrate it into `CI` (add a test in .github/actions/project-test/action.yml, use `xgo` run in dir `test\CI` directly before installing `spx` cmd tool) to facilitate validation of this solution during each PR. If CI passes, it's acceptable; if not, it's unacceptable.

joeykchen committed:
Add gop.mod
Refactor : Do Not Create A Temporary Directory And Walking It
Rename JSFuncOfWithError
Add SPX Runtime Version
Makefile Support Genarate Template
Remove xgo_autogen.go From Project Directory To Keep It Clean
Refactor Ensure GopMod
For Xgo Run

joeykchen commented:
> @joeykchen Better to integrate it into `CI` (add a test in .github/actions/project-test/action.yml, use `xgo` run in dir `test\CI` directly before installing `spx` cmd tool) to facilitate validation of this solution during each PR. If CI passes, it's acceptable; if not, it's unacceptable.

ok， after：
https://github.com/goplus/mod/pull/125
https://github.com/goplus/xgo/pull/2517

xushiwei commented on cmd/spxrun/runner/gop.mod:
```diff
@@ -0,0 +1,7 @@
+gop 1.3
```
rename `gop 1.3` => `xgo 1.3`
rename `gop.mod` => `gox.mod`

xushiwei commented on cmd/spxrun/main.go:
```diff
@@ -0,0 +1,46 @@
+
+	"github.com/goplus/spx/v2/cmd/spxrun/runner"
+)
+
```
是否还要考虑命令行参数支持，比如 -f 表示 full screen 等。
